setup.py

```diff
@@ -84,6 +84,9 @@ def scan_cpp(dir, files=[]):
 else:  # Linux, GCC
     extra_compile_flags += ["-std=c++17","-O3"]
 
+    machine_arch = os.uname()[-1]
+    if machine_arch == 'aarch64' or machine_arch.startswith('arm'):
+        extra_compile_flags += ["-DSEAL_BUILD_SEAL_C=1", "-DSEAL_USE_INTRIN=0"]

```

Comment:
Instead of modifying `extra_compile_flags`, could you try assigning the new define variables using `define_macros`? This way:
```python
    machine_arch = os.uname()[-1]
    if machine_arch == 'aarch64' or machine_arch.startswith('arm'):
        define_macros += [("SEAL_BUILD_SEAL_C", 1), ("SEAL_USE_INTRIN", 0)]
```
There should be no difference for the GCC compilation, but this way we can keep track of the custom defines at python level (otherwise it would stay generic).

Reply:
Better solve the other problem and then, when the test works, I will try with different syntax.